food-waste-express-api/routes/home.js

```diff
@@ -16,12 +16,48 @@ function getProduct(productId) {
     return productQuery.get(productId)
 }
 
+function getAllProductsInBasket(userBasket) {
+    return userBasket.get("basketOfProducts")
```

Comment:
Getting there! Still confused why we pass these strings as param?

Reply:
According to the Back4App documentation we're only able to get the value of the 
parameters by passing in an attribute in the form of a String. This is what I've been referencing when handling Parse, so if you see anything different please let me know!
<img width="1220" alt="Screen Shot 2022-08-01 at 11 18 57 PM" src="https://user-images.githubusercontent.com/68706299/182305334-82813bd9-fccb-461b-8988-c37e3d024f6e.png">

Reply:
I see, let's define a class and globally define the string variables there instead then